source/Calamari.Tests/AWS/CloudFormation/CloudFormationFixture.cs

```diff
@@ -41,157 +17,37 @@ public class CloudFormationFixture
         [Category(TestCategory.CompatibleOS.OnlyWindows)]
         public async Task CreateOrUpdateCloudFormationTemplate()
         {
-            var template = $@"
-{{
-    ""Resources"": {{
-        ""{StackName}"": {{
-            ""Type"": ""AWS::S3::Bucket"",
-            ""Properties"": {{ 
-                ""BucketName"": {StackName}
-            }}
-        }}
-    }}
-}}";
+            var templateFilePath = CloudFormationFixtureHelpers.WriteTemplateFile(CloudFormationFixtureHelpers.GetBasicS3Template(StackName));
+            var variables = new CalamariVariables();
+
             try
             {
-                DeployTemplate(StackName, template);
+                CloudFormationFixtureHelpers.DeployTemplate(StackName, templateFilePath, variables);
```

Comment:
You can prob just inline the instance of CalamariVariables here if you don't need to add anything specific into it (and remove changes on line 21)
```suggestion
                CloudFormationFixtureHelpers.DeployTemplate(StackName, templateFilePath, new CalamariVariables());
```